.zenodo.json

```
@@ -0,0 +1,29 @@
+{
+    "creators": [
+      {
+          "name": "Campolongo, Elizabeth G.",
+          "orcid": "https://orcid.org/0000-0003-0846-2413",
+          "affiliation": "The Ohio State University"
+      },
+      {
+          "name": "Radhakrishnan, Balaji",
+          "affiliation": "The Ohio State University"
```

```suggestion
          "name": "Radhakrishnan, Balaji",
          "affiliation": "The Ohio State University"
      },
      {
          "name": "Campolongo, Elizabeth G.",
          "orcid": "https://orcid.org/0000-0003-0846-2413",
          "affiliation": "The Ohio State University"
```
Should match citation order.